components/gravityforms/class-gf-convertkit.php

```diff
@@ -170,6 +171,29 @@ public function maybe_add_custom_field_mapping( $base_fields ) {
 		return $base_fields;
 	}
 
+	/**
+	 * If the connected account returns tags, add tags mapping to the feed settings
+	 * Otherwise, don't insert tags mapping
+	 *
+	 * @param array $base_fields
+	 *
+	 * @return array
+	 */
+	public function add_tags_mapping( $base_fields ) {
```

Comment:
Should be called `maybe_add_tags_mapping()` to follow current syntax of `maybe_add_custom_field_mapping()`

Reply:
++